blazor/dropdown-list/how-to/disable-the-dropdownlist-popup-items-based-on-selection.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Disable the DropDown List Popup items based on selection | Syncfusion
+description: Learn how to disable the DropDown List popup items based on the selection in the Syncfusion Blazor DropDown List component.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# How to disable the DropDown List popup items based on selection
+
+The following example demonstrates how to disable the DropDown List popup items based on the selection in the DropDown List component. This can be achieved by adding the e-disabled class to the selected list.
+
+```cshtml
+@using Syncfusion.Blazor
+@using Syncfusion.Blazor.DropDowns
+
+<SfDropDownList @ref="DropObj" ID="DDL" TValue="string" ShowClearButton="false" @bind-Value="@DropValue" TItem="Countries" Placeholder="e.g. Australia" DataSource="@Country">
+    <DropDownListFieldSettings Text="Name" Value="Code"></DropDownListFieldSettings>
+    <DropDownListEvents TValue="string" TItem="Countries" Opened="OnOpen"></DropDownListEvents>
+</SfDropDownList>
+
+@code {
+    public string DropValue { get; set; } = "BM";
```

Comment:
Use proper variable names and proper casing.

https://syncfusion.sharepoint.com/sites/Blazor/SitePages/C-%20Coding%20Best%20Practices.aspx